### PR TITLE
Triad bingo

### PR DESCRIPTION
**Pull request by crookedneighbor**
Companion for: https://github.com/HabitRPG/habitrpg/pull/4529

### CODE REVIEW

File: script/index.coffee [hunk 1]
```diff
@@ -774,6 +776,10 @@ api.wrap = (user, main=true) ->
           if not user.achievements.mountMasterCount
             user.achievements.mountMasterCount = 0
           user.achievements.mountMasterCount++
+          if giveTriadBingo
```
Comment (crookedneighbor) [inline script/index.coffee:779]: @deilann is this acceptable to you? 
- Set `giveTriadBingo = true`
- for every pet, check if `pet == -1`, and if so set `giveTriadBingo = false`. 
- if `giveTriadBingo == true` at the end, then increase count of triadBingoCount

You can't use `releaseBoth()` without having all 90 mounts, so there's no need to check for that.

**Comment by crookedneighbor** [comment]
When to add in tests for triadBingo, but it turns out that none of the tests for beastMaster and mountMaster actually work... `expect(user.achievements.beastMaster).to.be.ok` and `expect(user.achievements.beastMaster).to.not.be.ok` both evaluate true for some reason...
